### ISSUE TITLE
Discussion policies

### ISSUE THREAD

**Issue by raix** [MeteorCommunity/discussions#19]
Figure out how best to approach issues in the community

**Comment by raix** [comment]
We should also have a pattern for closing discussions and create a new issue with updated results in the description - eg. the original description with results added from the closed discussion? (we should lock the closed issue and guide traffic to the new issue)

This would also be the pattern where a discussion is really 2 or more issues in one?

**Comment by dandv** [comment]
Agree. Moved your flagging message to a new thread, #20. I think #14 is a good example of issue being kept on topic.

**Comment by mitar** [comment]
There is a discussion being happening about this here:
- https://github.com/meteor/meteor/pull/6171
- https://forums.meteor.com/t/how-open-source-meteor-really-is-if-pull-requests-are-not-getting-in/16483 (older thread)

**Comment by raix** [comment]
Looks good @mitar I might start creating pr's again
